packages/file-input/_file.scss

```diff
@@ -15,37 +15,65 @@
     --jkl-file-border-color-active: #{jkl.$color-snohvit};
 }
 
-@include jkl.comfortable-density-variables {
-    --jkl-file-padding: #{jkl.$spacing-16} #{jkl.$spacing-12};
-    --jkl-file-thumbnail-size: #{jkl.rem(64px)};
-    --jkl-file-gap: #{jkl.$spacing-16};
-}
-
-@include jkl.compact-density-variables {
-    --jkl-file-padding: #{jkl.$spacing-12} #{jkl.$spacing-8};
-    --jkl-file-thumbnail-size: #{jkl.rem(44px)};
-    --jkl-file-gap: #{jkl.$spacing-8};
-}
-
 .jkl-file {
     @include jkl.motion("standard", "snappy");
+
+    @include jkl.comfortable-density {
+        --jkl-file-padding: #{jkl.$spacing-24} #{jkl.$spacing-24};
+        --jkl-file-thumbnail-size: #{jkl.rem(64px)};
+        --jkl-file-gap: #{jkl.$spacing-16};
+        --jkl-file-font-size: #{jkl.rem(20px)};
+        --jkl-file-line-height: #{jkl.rem(32px)};
+    }
+
+    @include jkl.compact-density {
+        --jkl-file-padding: #{jkl.$spacing-12} #{jkl.$spacing-12};
+        --jkl-file-thumbnail-size: #{jkl.rem(44px)};
+        --jkl-file-gap: #{jkl.$spacing-8};
+        --jkl-file-font-size: #{jkl.rem(16px)};
+        --jkl-file-line-height: #{jkl.rem(24px)};
+    }
+
     transition-property: background-color, border-color, color;
 
     color: var(--jkl-file-color);
     border: 1px var(--jkl-file-border-color) solid;
-    padding: var(--jkl-file-padding);
     margin: 0;
     min-width: jkl.rem(240px);
 
     display: flex;
     flex-flow: nowrap row;
-    gap: var(--jkl-file-gap);
-    align-items: center;
 
     @include jkl.text-style("small");
     @include jkl.reset-outline;
 
-    a {
+    &__content {
+        display: flex;
+        flex-basis: 100%;
+        gap: 1em;
+        background-color: var(--jkl-color-background-container-high);
+        padding: var(--jkl-file-padding);
+        border-radius: jkl.rem(2px);
+        box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.05);
+
+        &--error {
+            background-color: var(--jkl-error);
+        }
+
+        &--warning {
+            background-color: var(--jkl-warning);
+        }
+
+        &--error,
+        &--warning {
+            --jkl-color-text-default: jkl.$color-granitt;
+
+            color: jkl.$color-granitt;
```

Comment:
Her kan du bruke `var(--jkl-color-text-on-alert)` for å få en farge som passer i både dark og light mode 😊

Jeg tror generelt det er en god idé å få gjort om fargevariablene fra øverst i fila til bruk av de semantiske fargevariablene våre. Kan godt se på det sammen hvis du vil!